doc/man3/EVP_PKEY_keygen.pod

```diff
@@ -87,13 +91,25 @@ and retrieve an opaque pointer. This can be used to set some application
 defined value which can be retrieved in the callback: for example a handle
 which is used to update a "progress dialog".
 
+EVP_PKEY_Q_keygen() abstracts from the explicit use of B<EVP_PKEY_CTX> while
+providing a 'quick' but limited way of generating a new asymmetric key pair.
+It is used to implement shorthands for the most common cases of key generation.
+As usual, the library context I<libctx> and property query I<propq>
+can be provided fetching algorithms from providers.
+If I<type> is C<RSA>, the following unsigned integer parameter is used to
+specify the size of RSA keys.
+If I<types> is C>EC>, the I<name> is used to specify the name of an EC curve.
```

Comment:
<>
I thought we used B<> in other places?

Reply:
Function parameters were formerly marked by `B< >` but nowadays we use `I< >`,
while for type names like `EVP_PKEY_CTX` we use `B< >`.

Reply:
The `C>EC>` looks wrong.

Reply:
Yes, I also noticed this, fixed.